### PR TITLE
Add support of stored procedure

### PR DESCRIPTION
**Pull request by leilifremont**
Tested by dbValidatorTest

### CODE REVIEW

File: pkg/gencosmosdb/cosmosdb/storedprocedure.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
+package cosmosdb
+
+import (
+	"context"
+	"net/http"
+)
+
+// StoredProcedure represents a stored procedure
+type StoredProcedure struct {
+	ID         string `json:"id,omitempty"`
```
Comment (julienstroheker) [inline pkg/gencosmosdb/cosmosdb/storedprocedure.go:10]: Can you add this description at the top :

"ID is the user generated unique name for the stored procedure. No two stored procedures can have the same IDs."

File: pkg/gencosmosdb/cosmosdb/storedprocedure.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	ResourceID string `json:"_rid,omitempty"`
+	Timestamp  int    `json:"_ts,omitempty"`
+	Self       string `json:"_self,omitempty"`
+	ETag       string `json:"_etag,omitempty"`
+	Body       string `json:"body,omitempty"`
```
Comment (julienstroheker) [inline pkg/gencosmosdb/cosmosdb/storedprocedure.go:15]: Same please add :

"Body of the stored procedure."

**Review by julienstroheker — CHANGES_REQUESTED** [review]
Tiny details

File: pkg/gencosmosdb/cosmosdb/storedprocedure.go (hunk 1 above)
Comment (leilifremont) [reply pkg/gencosmosdb/cosmosdb/storedprocedure.go:11]: Done!

**Review by julienstroheker — APPROVED** [review]

**Pull request merged**